prow/cluster/jobs/istio/test-infra/istio.test-infra.master.gen.yaml

```diff
@@ -196,7 +196,7 @@ postsubmits:
     cluster: test-infra-trusted
     decorate: true
     labels:
-      preset-prow-deployer-service-account: "true"
+      preset-service-account: "true"
```

Comment:
This doesn't seem quite right. The `service-account` secret's SA shouldn't have permission to upload images? I guess the trusted cluster could have a different identity in the `service-account` secret, but this smells like either confusing naming or incorrect credentials.
/hold

Reply:
The SA is `istio-prow-test-job@istio-testing.iam.gserviceaccount.com` and it is K8s admin, Storage owner (among other things). I will follow-up and create a `preset-prow-pusher-service-account` SA for push jobs. It will be a challenge for another day to update the SA in the build cluster.

Reply:
Will you a) please use workload identity instead of a service account? b) not use a preset?

Reply:
> a: use workload identity
Workload identity is not something I have prioritized (or learned) yet. It is being tracked in: https://github.com/istio/test-infra/issues/2282 and recently labeled as _help wanted_. It should probably not be a prerequisite for fixing this one misconfigured job - since it will impact other jobs and clusters as well; unless, of course, there is a way to progressive rollout the change for this one job without affecting the others?

> b: do not use preset
I can "resolve" the value of this preset (and directly add the labels,  mounts, etc.) instead of actually using the preset.

Reply:
/unhold
Support for preset resolution and using "pusher" SA in: https://github.com/istio/test-infra/pull/2508